src/scss/initial/_colors.scss

```diff
@@ -23,9 +26,9 @@
 
 /// @group init-colors
 [data-ccs-colors] {
-  @include tools.tokens--(config.$ccs-colors);
-  @include tools.tokens--(config.$background-colors);
-  @include tools.tokens--(config.$content-colors);
+  @include config.tokens--(ccs-defaults.$ccs-colors);
```

Comment:
If we break the color partials into individual variables, then we can use `meta.module-variables` here to pass along a map of those variables. Otherwise, we don't seem to be actually making use of `sass:meta` in this file yet.